src/solvers/dg/2d/dg.jl

```diff
@@ -1530,43 +1563,152 @@ end
 
 
 """
-    calcflux_fv!(fstar1, fstar2, u_leftright, u, element_id, dg::Dg2D)
+    calcflux_fv!(fstar1_L, fstar1_R, fstar2_L, fstar2_R, u_leftright, u, element_id, dg::Dg2D)
```

Comment:
```suggestion
    calcflux_fv!(fstar1_L, fstar1_R, fstar2_L, fstar2_R, u_leftright, u, element_id, dg::Dg2D, nonconservative_terms)
```
To match the function signature?